File: source/downloads/code/LeetCode/Python/Bitwise-and-of-Numbers-Range.py
class Solution(object):
    def rangeBitwiseAnd(self, m, n):
        """
        :type m: int
        :type n: int
        :rtype: int
        """
        p = 0
        while m!=n:
            m = m>>1
            n = n>>1
            p+=1
        return m<<p

File: source/downloads/code/LeetCode/Java/Bitwise-and-of-numbers-range.java
public class Solution {
    public int rangeBitwiseAnd(int m, int n) {
        int p = 0;
        // 有p位是相同的
        while (m!=n){
            m = m >> 1;
            n = n >> 1;
            p ++;        
        }
        return m << p;
    }
}
